eng/templates/Azure.Management.Template/src/Azure.Management.Template.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
While I don't mind a second template I'd like to understand if there is a good reason for it. Can we converge this with the existing template?

Reply:
Are you referring to the SDK\Template folder? Talked to Pavel, the intended purpose is different thus the file content etc. So keeping a separate one. Also that template folder was not set up as dotnet new template.